### ISSUE TITLE
When making a server v9.999999 crashes the server

### ISSUE THREAD

**Issue by Mortadelyfile** [abysschroma/NTM-but-uncomfortable#1]
Whenn making a server using v9.999999 the server crashes, this odes not occur when using lastest verion of normal HBM's NTM Extended

**Comment by abysschroma** [comment]
Yeah expected
I was busy for my school work till 15, and so i never tested it in server. I’ll test it…sometime
Im guessing the server is trying to perform an action only allowed on clients or smth, but idrk as aforementioned never really tried 

Btw happy first Issue on this chaotic fork lol

**Comment by abysschroma** [comment]
Id be happy if you provided your crash log because my PC drive’s gonna implode lol

**Comment by Mortadelyfile** [comment]
[crash-2024-12-15_16.15.23-server (1).txt](https://github.com/user-attachments/files/18211347/crash-2024-12-15_16.15.23-server.1.txt)
That's the log, I must note that this was a friend server so I don't have anymore files, but I can recreate the problem if needed.

**Comment by Mortadelyfile** [comment]
Also, does any version work on servers?

**Comment by abysschroma** [comment]
> Also, does any version work on servers?

Not sure since i Literally Never Tested It On Server :skull:

**Comment by abysschroma** [comment]
Seems like it's there since v6.

**Comment by abysschroma** [comment]
OK IT WAS SUPER DUMB ISSUE

BASICALLY IDFK WTF I WAS THINKING BUT I HAD PUT AN COMPLETELY UNNECESSARY CLIENT-ONLY ACCESS RESTRICTION TO A VALUE AND THAT WAS CAUSING THE ISSUE :sKULL:

**Comment by abysschroma** [comment]
Ok, theres still countless issues on servers but at least crashing on startup is now fixed in v10-6.192853E+19

**Comment by Mortadelyfile** [comment]
Magnificient, do you know aprox. when v10-6.192853E+59 will be released?

**Comment by abysschroma** [comment]
not at all, there's reason literally all the releases are labelled pre-release lol
ill give u dev build if u really want, but keep in mind it has many unfinished feature that confuses the hella off lol (and also has NTM structure generation temporarily disabled because it keeps crashing the game)

**Comment by abysschroma** [comment]
espicially since the entire thing is dragged by pwr which's so far taking 2 months

**Comment by Mortadelyfile** [comment]
A dev release would be good, it's just for friends and I creative server to test HBM's things, not need for polishment.

**Comment by abysschroma** [comment]
so, do u want one?

**Comment by Mortadelyfile** [comment]
Yes, thx

**Comment by abysschroma** [comment]
https://drive.google.com/file/d/12lgSTOCwa2Qtg_cBLmWKXb4b-JUjmumS/view?usp=drive_link

**Comment by Mortadelyfile** [comment]
Thank You
